### PR TITLE
Hit policies and satisfied rules

### PR DESCRIPTION
**Pull request by tiruis**
Hit policy Unique only will return one satisfied rule and others policies will return more than one

### CODE REVIEW

File: content/reference/dmn11/decision-table/hit-policy.md [hunk 1]
```diff
@@ -65,8 +65,8 @@ supported by the Camunda DMN engine:
 
 A hit policy specifies how many rules of a decision table can be satisfied and
 which of the satisfied rules are included in the [decision table result]. The
-hit policies [Unique], [Any] and [First] will always return a maximum of one
-satisfied rule. The hit policies [Rule Order] and [Collect] can return
+hit policies [Unique] will always return a maximum of one
+satisfied rule. The hit policies [Any], [First], [Rule Order] and [Collect] can return
```
Comment (yanavasileva) [inline content/reference/dmn11/decision-table/hit-policy.md:69]: `First` will return also a single result
```suggestion
hit policies [Unique] and [First] will always return a maximum of one
satisfied rule. The hit policies [Any], [Rule Order] and [Collect] can return
```

**Comment by yanavasileva** [comment]
Hi @tiruis,

Thank you for your contribution and welcome to the community!
Please check my note in the comment above.

Best regards,
Yana

**Comment by tiruis** [comment]
Hi,

Para1
    The Role of a Hit Policy
A hit policy specifies how many rules of a decision table can be satisfied and which of the satisfied rules are included in the decision table result<https://docs.camunda.org/manual/7.12/user-guide/dmn-engine/evaluate-decisions/#interpret-the-dmndecisiontableresult>. The hit policies Unique<https://docs.camunda.org/manual/7.12/reference/dmn11/decision-table/hit-policy/#unique-hit-policy>, Any<https://docs.camunda.org/manual/7.12/reference/dmn11/decision-table/hit-policy/#any-hit-policy> and First<https://docs.camunda.org/manual/7.12/reference/dmn11/decision-table/hit-policy/#first-hit-policy> will always return a maximum of one satisfied rule. The hit policies Rule Order<https://docs.camunda.org/manual/7.12/reference/dmn11/decision-table/hit-policy/#rule-order-hit-policy> and Collect<https://docs.camunda.org/manual/7.12/reference/dmn11/decision-table/hit-policy/#collect-hit-policy> can return multiple satisfied rules.
Tiru >> Here it says Unique, Any and First will return a maximum of one satisfied rule.

Para2
Any Hit Policy

Multiple rules can be satisfied. However, all satisfied rules must generate the same output. The decision table result contains only the output of one of the satisfied rules.

If multiple rules are satisfied which generate different outputs, the hit policy is violated.

Para2
First Hit Policy

Multiple rules can be satisfied. The decision table result contains only the output of the first satisfied rule.


Tiru >> From the above two para we can conclude that Any and First will return multiple satisfied rules.
Statement in para1 is contradicting with para2 and para3
Regards,
Tiru
+917397379390

**Comment by tiruis** [comment]
Please tell whether my defect is valid or invalid

File: content/reference/dmn11/decision-table/hit-policy.md (hunk 1 above)
Comment (yanavasileva) [reply content/reference/dmn11/decision-table/hit-policy.md:69]: Please disregard the above comment.

**Comment by yanavasileva** [comment]
Hi @tiruis,

I am sorry, my first comment was wrong, I had to double-check in the DMN Specification:
https://www.omg.org/spec/DMN/1.3/PDF

Please go to Section **8.2.10 Hit policy** (or search for the string `Single hit policies` in the PDF).
There you can find that `Unique`, `Any`, and `First` are defined as `Single hit policies` and they have a more detailed explanation of the policies.

What that being said, I don't think that the documentation needs to be changed as per the official DMN Specification. Hereby I am closing the PR.

Best regards,
Yana

**Pull request closed without merging**